test/unit/saveArticles.test.ts

```diff
@@ -238,3 +238,16 @@ test('--customFlavour', async (t) => {
     t.ok(ParisDocument.querySelector('#PRE_PROCESSOR'), `Paris was correctly pre-processed`);
     t.ok(PragueDocument.querySelector('#POST_PROCESSOR'), `Prague was correctly post-processed`);
 });
+
+test('Test Subtitles', async(t) => {
+    const { downloader, mw, dump } = await setupScrapeClasses({ format: '' });
+
+    // Wikicode is taken from article "Mechanical energy" which has a video with subtitle
+    const wikicode = `==Conservation of mechanical energy==
+    [[File:Physicsworks.ogv|thumb|200px|alt="Lecture demonstrating conservation of mechanical energy"|MIT professor [[Walter Lewin]] demonstrating conservation of mechanical energy]]`;
+
+    const htmlDoc = domino.createDocument(await convertWikicodeToHtml(wikicode, dump));
+    const contentRes = await treatSubtitles(htmlDoc.querySelector('track'), downloader, 'en.wikipedia.org', mw);
+
+    t.ok(contentRes, 'Successfully downloaded the subtitles');
```

Comment:
* We should have in test a function which just takes as argument `wiki`, `html` `comment` and make the comparison.
* Never write "Successfully" or similar in a test comment. You could write it everywhere and this is useless and in particular if it fails. Just write in a few words what the test does like for example "video with subtitle rewriting".